docs/newbs_building_firmware.md

```diff
@@ -59,6 +59,22 @@ Checking file size of planck_rev5_default.hex
  * The firmware size is fine - 27312/28672 (95%, 1360 bytes free)
 ```
 
+## Configure Your Build Environment (Optional)
```

Comment:
This might be better moved to the start of this page? There's earlier references to configuring your environment.